android/java/org/chromium/chrome/browser/settings/BraveSyncScreensPreference.java

```diff
@@ -1517,6 +1518,8 @@ private void setSyncDoneLayout() {
             mDeviceInfoObserverSet = true;
         }
 
+        BravePermissionUtils.showNotificationPermissionDialog(getActivity());
```

Comment:
should we somehow explain why are we requesting the permission? or I can't find where it happens.

Reply:
@AlexeyBarabash we are showing system permission dialog for notification. when user `Send to your devices` option from other device in the sync chain, we should get a notification with the tab url. without the permission, we won't get the notification.

Reply:
DMed and resolved